tasks.py

```diff
@@ -6,6 +7,9 @@
 SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
 TARGET_CONTAINER_NAME = 'yaksh_code_server'
 SRC_IMAGE_NAME = 'fossee/yaksh_codeserver'
+CHECK_FILE = 'touch_me_not.txt'
```

Comment:
Maybe this can be be 'server_running.txt'

Reply:
Fixed